nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/util/FlowParser.java

```diff
@@ -278,13 +278,14 @@ private List<PortDTO> getPorts(final Element element, final String type) {
      * @param flowDocument flowDocument of the associated XML content to write to disk
      * @param flowXmlPath  path on disk to write the flow
      * @throws IOException if there are issues in accessing the target destination for the flow
-     * @throws TransformerException if there are issues in the xml transformation process
      */
-    public void writeFlow(final Document flowDocument, final Path flowXmlPath) throws IOException, TransformerException {
+    public void writeFlow(final Document flowDocument, final Path flowXmlPath) throws IOException {
```

Comment:
I was going to ask about caller handling of the unchecked exception, but it doesn't look like this is in use?

Reply:
Correct, the caller would receiving a runtime XML `ProcessingException`, instead of the checked `TransformerException`. As mentioned, this does not appear to be an issue as this method does not appear to be used.